index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Challenge - Ian Grantham</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- overall grid -->
+    <section class="grid">
+
+        <!-- LH menu bar -->
+        <section class="left">
+
+            <section class="menu">
+                <div class="logo">
+                    <svg id="twitter_badge" xmlns="http://www.w3.org/2000/svg"  
+                    viewBox="0 0 333333 333333" 
+                    shape-rendering="geometricPrecision" 
+                    text-rendering="geometricPrecision" 
+                    image-rendering="optimizeQuality" 
+                    fill-rule="evenodd" clip-rule="evenodd">
+                        <path d="M333328 63295c-12254 5480-25456 9122-39241 10745 14123-8458 24924-21861 30080-37819-13200 
+                        7807-27871 13533-43416 16596-12499-13281-30252-21537-49919-21537-37762 0-68336 30591-68336 68330 0 
+                        5326 591 10537 1748 15562-56820-2880-107194-30081-140915-71467-6049 10435-9250 22300-9250 34367v8c0 
+                        23696 12031 44654 30389 56876-11202-333-21739-3457-30991-8519v854c0 33138 23554 60789 54852 67039-5734 
+                        1557-11787 2417-18023 2417-4417 0-8673-455-12905-1224 8742 27139 33975 46923 63857 47500-23430 
+                        18356-52858 29286-84939 29286-5537 0-10931-339-16318-984 30326 19458 66251 30727 104844 30727 
+                        125735 0 194551-104198 194551-194543 0-3002-67-5911-191-8852 13354-9553 24932-21609 34097-35333l31-31-6 
+                        4z" fill="#1da1f2"/>
+                    </svg>
```

Comment:
your use of different semantic tags. I definitely will implement this with my code, mine was mainly divs